README.md

```diff
@@ -29,7 +29,11 @@ If your heart is set on the bleeding edge, install from source. However, please
 #### Dependencies
 Building geth requires both Go >=1.8 and a C compiler.
 
+#### Download/clone the source code
```

Comment:
Let's reword this to `Get source and dependencies`, since it's not `clone`ing (in the __git__ sense) and it'll grab more than just the source (the `/...` tells go to _get_ all the package's required dependencies as well).